geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -235,6 +237,12 @@ AtomicLong5 getEventSeqNum() {
     return eventSeqNum;
   }
 
+  boolean notPrimary = true;
+  volatile boolean allChildBucketsBecomePrimary = false;
+  private Object allChildBucketsBecomePrimaryLock = new Object();
```

Comment:
Why are some of these members private and others not?

Reply:
non private members are accessed in the unit test.